src/sagemaker/mxnet/estimator.py

```diff
@@ -17,27 +17,158 @@
 from sagemaker.mxnet.defaults import MXNET_VERSION
 from sagemaker.mxnet.model import MXNetModel
 
+from sagemaker.utils import get_config_value
+
+
+
+import contextlib
+import logging
+import os
+import shutil
+import subprocess
+import tempfile
+import threading
+import time
+
+logging.basicConfig()
+LOGGER = logging.getLogger('sagemaker')
+
+class Tensorboard(threading.Thread):
```

Comment:
It if actually the same code, there is only a minor difference in the log string on line 143 in the document you reference.

Reply:
I see -- we'd like to avoid duplicating code since that'll make this harder to maintain. Would it be possible to refactor this so that both TF and MXNet use the same code without having it duplicated?